docs/operating-scylla/procedures/cluster-management/remove-node.rst

```diff
@@ -2,13 +2,34 @@
 Remove a Node from a ScyllaDB Cluster (Down Scale)
 ***************************************************
 
+.. caution::
+
+    The procedure described below applies to clusters where Raft-based topology 
+    is enabled. Raft-based topology is automatically enabled in new clusters.
+
+    If you've upgraded an existing cluster from version 5.4, ensure that you 
+    :doc:`manually enabled Raft-based topology </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-raft-topology>`.
+    Without Raft-based topology enabled, you must consider the following limitations 
+    while following the procedure:
+    
+    * It’s essential to ensure the removed node will **never** come back to the cluster, which might adversely affect 
+      your data (data resurrection/loss). To prevent the removed node from rejoining the cluster, remove that node 
+      from the cluster network or VPC.
+    * You can only remove one node at a time. You need to verify that the node has been removed before removing another one.
+    * If ``nodetool decommission`` starts executing but fails in the middle, for example, due to a power loss, consult the 
+      :doc:`Handling Membership Change Failures </operating-scylla/procedures/cluster-management/handling-membership-change-failures>`
+      document. 
+
 You can remove nodes from your cluster to reduce its size.
 
-.. note::
+Prerequisites
+-------------
+
+Adding a new node requires at least a quorum of nodes in a cluster to be available. 
```

Comment:
Removing

Reply:
In fact, `decommission` requires ALL nodes to be available. (This limitation was always present and remains.)

`removenode` requires quorum, and the only dead nodes can be the removed node and those passed to --ignore-dead-nodes